cvrf2csaf/section_handlers/comment.py

```diff
@@ -0,0 +1,13 @@
+import logging
+from ..common.common import SectionHandler
+
+
+class Comment(SectionHandler):
+    def __init__(self):
+        super().__init__()
+
+    def _process_mandatory_elements(self, root_element):
+        logging.warning(f'Not implemented input {self.__class__}. No parser written.')
+
+    def _process_optional_elements(self, root_element):
+        pass
```

Comment:
It is unclear to me, what this is used for. Is there an example you can share?

Reply:
As agreed above, `comment` has been in some of the examples. Removed class.